plans/tier0.fmf

```diff
@@ -0,0 +1,43 @@
+discover+:
+    tier: 0
+
+/changed_yum_conf:
+  discover+:
+    test: changed-yum-conf
+
+/check_custom_repo:
```

Comment:
We need to add the preparation role that adds custom repos on the target machine